### DISCUSSION TITLE
Joomla Installation > Vagrant

### DISCUSSION THREAD

**Discussion by mauritza** [joomlatools/joomla-vagrant#181] [Q&A]
Hi Johan,

Thanks for the swift response via joomlatools support, much appreciated !!

As mentioned I am a total newbie to Joomla & Vagrant, I have however a fair amount of experience with Linux in general.

I have gone through the initial setup of vagrant and Virtual Box (from the below link), when running **vagrant up** the below is outputted.

![image](https://user-images.githubusercontent.com/43013143/126566837-521d98e1-5c49-4bde-8db1-f93886abd472.png)

Also:  

- I haven't altered the downloaded Vagrantfile
- I have added the details on slid 5 to my host file.

https://www.joomlatools.com/developer/vagrant

Many thanks!

**Comment by johanjanssens** [comment]
Hello @mauritza, this error indicates that your host (the machine you are using to setup Vagrant on) doesn't support NFS. NFS is uses to share files and keep them synced between the Vagrant box and the host.

What operating system are you using to setup the Vagrant?

**Reply by mauritza** [reply]
I am using Windows 10 as my host machine & WSL as the subsystem for vagrant --- VirtualBox as the provider 

Any suggestions on how then I am able to have Joomla install with the neccessaries like a Vagrant box?

https://www.joomlatools.com/developer/vagrant

**Reply by johanjanssens** [reply]
Sorry for the delay in getting back to you. If you have issues with NFS please check: https://www.joomlatools.com/developer/vagrant/faq#how-to-troubleshoot-nfs-mount-issues
